docs/tutorials/uniq-factories/factory-management/uniq-first-hand-purchase-example-project.md

```diff
@@ -0,0 +1,54 @@
+---
+title: 'Uniq First-Hand Purchase Example Project'
+
+order: 4
+---
+
+
+# Uniq First-Hand Purchase Example Project
+
+This page covers an example project that lets users buy, swap or exchange their Uniqs using first hand purchase functionality. For installation instructions refer to the repository itself.
+
+Code is available here: https://github.com/ultraio/example-first-hand-purchase
+
+## Use cases
+
+- This project provides a reference implementation for integrating first hand purchase functionality into your web-based solution.
+
+- Parts of the implementation can be extracted and reused as needed or you can simply update the code to your needs and deploy it as is.
+
+- If your project requires users to be able to get Uniqs in any way without having a backend then this example project can benefit you as it operates as a static website and interacts with Ultra Blockchain directly.
+
+- Since some of the settings of token factories are fixed during creation and cannot be changed you could utilize first-hand purchase functionality to let users exchange their token from existing factory to a new factory. This new factory could have different values for immutable settings (such as transferability window or tradeability window) and by exchanging a token you will effectively change those values.
```

Comment:
Let's change the term: "token" with "uniq"? since we are trying to train end-users that is our term?

Reply:
Right, sorry, somehow forgot about that